contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1271,17 +1271,31 @@ select
       when 'p' then 'P'::varchar(2)
       when 'a' then 'AF'::varchar(2)
       else
-        case format_type(p.prorettype, null) when 'trigger'
-          then 'TR'::varchar(2)
+        case 
+          when format_rettype = 'trigger'
+            then 'TR'::varchar(2)
+          when func_results LIKE 'TABLE(%' then
+            case 
+              when format_rettype = 'record'
```

Comment:
I think we could call function here to check if it is 'record' or use 'record''s OID

Reply:
Decided to not use the 'record' field due to the indistinguishable case you mentioned.